frontend/express/public/javascripts/countly/vue/components/input.js

```diff
@@ -811,6 +813,25 @@
                     return true;
                 }
                 return Array.isArray(this.innerValue) && this.innerValue.length >= this.minItems && this.innerValue.length <= this.maxItems;
+            },
+            isSearchShown: function() {
+                if (this.searchable) {
+                    if (this.remote) {
```

Comment:
We can wait for remote data to resolve, and then mark searchable as true/false.

Reply:
That is actually already reflected in options/flatoptions.
The problem here is that we need to keep the initial amount of options. I'll see what I can do with that

Reply:
Updated